sqlflow/client.py

```diff
@@ -0,0 +1,69 @@
+import os
+import logging
+import grpc
+import google.protobuf.wrappers_pb2 as wrapper
+
+import sqlflow.proto.sqlflow_pb2
+import sqlflow.proto.sqlflow_pb2_grpc
+
+_LOGGER = logging.getLogger(__name__)
+
+class Client(object):
+    def __init__(self, server_url=None):
+        """A minimum client that issues queries to and fetch results/logs from sqlflowserver.
+
+        Args:
+            server_url(str):
+                sqlflowserver url
```

Comment:
Need doc to specify default None means read value from environment variable SQLFLOW_SERVER